katello/katello-debug.sh

```diff
@@ -13,6 +13,22 @@ then
   exit 1
 fi
 
+# foreman-debug will truncate any file beyond 5k lines
+# for larger files we need to copy the entire file.
+copy_files() {
+  for FILE in $*; do
+    echo "Copying entire file: $FILE"
+    if [ \( -f "$FILE" -o -h "$FILE" \) -a \( -r "$FILE" -a -s "$FILE" \) ]; then
+      printv " - $FILE"
+      SUBDIR=$(dirname $FILE)
+      [ ! -d "$DIR$SUBDIR" ] && mkdir -p "$DIR$SUBDIR"
+      cp "$FILE" "$DIR$FILE"
+    fi
+  done
+}
```

Comment:
curious if we've discussed possibly changing this behavior as it can be really annoying for files it captures